.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea/*
-*__pycache__*
\ No newline at end of file
+*__pycache__*
+dataset*.pt
+tmp/*
```

Comment:
You still should add this folder to the repo :)

Reply:
sry, I forgot to force add .keep